backend/src/main/java/com/woowacourse/levellog/prequestion/domain/PreQuestion.java

```diff
@@ -0,0 +1,60 @@
+package com.woowacourse.levellog.prequestion.domain;
+
+import com.woowacourse.levellog.common.domain.BaseEntity;
+import com.woowacourse.levellog.common.exception.InvalidFieldException;
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.member.domain.Member;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class PreQuestion extends BaseEntity {
+
+    @ManyToOne
+    @JoinColumn(nullable = false, foreignKey = @ForeignKey(name = "fk_pre_question_levellog"))
+    private Levellog levellog;
+
+    @ManyToOne
+    @JoinColumn(nullable = false, foreignKey = @ForeignKey(name = "fk_pre_question_from_member"))
+    private Member from;
+
+    @Lob
+    private String preQuestion;
+
+    public PreQuestion(final Levellog levellog, final Member from, final String preQuestion) {
+        validateIsAuthor(levellog, from);
+        validateQuestion(preQuestion);
+
+        this.levellog = levellog;
+        this.from = from;
+        this.preQuestion = preQuestion;
+    }
+
+    private void validateIsAuthor(final Levellog author, final Member member) {
+        if (author.isAuthor(member)) {
+            throw new InvalidFieldException("내 글의 사전 질문을 작성할 수 없습니다.");
+        }
+    }
+
+    private void validateQuestion(final String question) {
```

Comment:
```suggestion
    private void validatePreQuestion(final String question) {
```
현재 `인터뷰질문`과 `사전인터뷰` 도메인 용어가 혼용될 수 있으니 비록 특정 도메인 내부라고 하더라도 정확한 용어를 사용하는게 헷갈리지 않을거 같아요!